plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -71,6 +74,17 @@ export class GraphQLInstrumentation extends InstrumentationBase {
       VERSION,
       Object.assign({}, DEFAULT_CONFIG, config)
     );
+    const ignoreMap = new Map();
+    config.ignoreFields?.map(item => ignoreMap.set(item, true));
+    this._setIgnoreMap(ignoreMap);
+  }
+
+  private _getIgnoreMap(): Map<string, boolean> {
+    return this._ignoreMap;
+  }
+
+  private _setIgnoreMap(map: Map<string, boolean> = new Map()) {
+    this._ignoreMap = map;
```

Comment:
this whole function can be removed

Reply:
It is now in two locations for the set behavior, so I kept this but repurposed